Source/WebCore/page/writing-tools/WritingToolsTypes.h

```diff
@@ -111,6 +111,15 @@ struct TextSuggestion {
     State state;
 };
 
+#pragma mark - Animations
+
+enum class RunAnimation : uint8_t {
+    DoNotRun,
+    OnlyReplaceText,
+    RunAnimation
```

Comment:
Strange to duplicate the enum name in the value. I would use `No`/`Yes`.

Reply:
I changed the name to RunMode, and I don't think that no/yes are descriptive enough.